packages/dht/src/dht/DhtNode.ts

```diff
@@ -112,24 +117,62 @@ const logger = new Logger(module)
 
 export type Events = TransportEvents & DhtNodeEvents
 
-export const createPeerDescriptor = (msg?: ConnectivityResponse, nodeId?: DhtAddress): PeerDescriptor => {
+const calculateNodeIdRaw = (signingModule: ISigningModule, ipAddress: number,
```

Comment:
We could instantiate new `EthereumSigningModule` here and not give this as a parameter. Maybe could even inline the methods `EthereumSigningModule`, and just call methods of `@streamr/utils`.

The `ISigningModule` is not needed as we support only signing type (when we'll add the support for multiple signing types, we'll most likely add a new signatureType to `PeerDescriptor`)